wazo_calld/plugins/calls/services.py

```diff
@@ -443,6 +445,7 @@ def channel_destroyed_event(ari, event):
         call.line_id = channel_variables.get('WAZO_LINE_ID') or None
         call.creation_time = channel.get('creationtime')
         call.answer_time = channel_variables.get('WAZO_ANSWER_TIME') or None
+        call.is_autoprov = event['channel']['dialplan']['context'] == AUTOPROV_CONTEXT
```

Comment:
For good styling, shouldn't this attribute also be added to Call helper class in calls/call.py?

Reply:
yep, done